.travis.yml

```diff
@@ -1,66 +1,21 @@
 sudo: required
 
+language: c
+
 services:
     - docker
-    
+
 cache: ccache
 
 jobs:
     include:
-        - stage: build docker image
-          env:
-              # The machine id will be passed to Dockerfile for later checks
-              - MACHINE_ID=$(cat /var/lib/dbus/machine-id)
-          before_script: &update
-              # Ensure the latest version of docker is installed
+        - stage: coverity scan
+          compiler: gcc
```

Comment:
Given that everything happens inside a docker container built with `build-docker-image.sh`, I doubt `language: c` and `compiler: gcc` are necessary.